front/src/modules/pathfinding/hooks/usePathfinding.ts

```diff
@@ -225,8 +236,24 @@ export const usePathfinding = (
 
         try {
           const pathfindingResult = await postPathfindingBlocks(pathfindingInput).unwrap();
+          if (pathfindingResult.status === 'invalid_path_items') {
+            const invalidPathItems = pathfindingResult.items
+              .map((item) => {
+                return 'trigram' in item.path_item ? item.path_item.trigram : null;
```

Comment:
Can you add a comment TODO here to precise that we currently handle only invalid pathSteps with trigram ? Later we will have to handle invalid pathSteps with trackOffset, opId and uic

Reply:
done